tgui/packages/tgui/interfaces/TelecommsMonitor.tsx

```diff
@@ -17,157 +19,204 @@ enum Screen {
   Machine,
 }
 
+type Machine = {
+  id: string;
+  name: string;
+  icon: string;
```

Comment:
```suggestion
  id: string | null;
  name: string | null;
  icon: string | null;
```

Reply:
Those vars can't be `null` and if they will become such, this means somebody messed something and UI should break.